app/models/entitlement/subscription_entitlement_privilege.rb

```diff
@@ -22,7 +22,14 @@ class SubscriptionEntitlementPrivilege
     attribute :sub_entitlement_value_id, :string
 
     def config
-      JSON.parse(super)
+      v = super
+      JSON.parse(v) if v.is_a?(String)
```

Comment:
Not a blocker :)

```suggestion
      super.tap { JSON.parse(it) if it.is_a?(String) }
```

Also, I don't know if is is expected but if config is not a string, the method will return `nil`